app-core.js

```diff
@@ -151,14 +122,38 @@ async function serveOctoFarmNormally(app, port) {
   app.get("*", function (req, res) {
     res.redirect("/");
   });
+}
+
+async function serveOctoFarmNormally(app, quick_boot = false) {
+  if (!quick_boot) {
+    logger.info("Initialising FarmInformation...");
+    await PrinterClean.initFarmInformation();
+
+    logger.info("Initialising Client Settings...");
+    await ClientSettings.init();
+
+    logger.info("Initialising OctoFarm State...");
+    const { Runner } = require("./server_src/runners/state.js");
+    const stateRunnerReport = await Runner.init();
+    logger.info("OctoFarm State returned", stateRunnerReport);
 
-  return listenerHttpServer;
+    logger.info("Initialising SystemRunner...");
+    const sr = await SystemRunner.init();
+    logger.info("OctoFarm SystemRunner returned", sr);
+
+    await optionalInfluxDatabaseSetup();
+  }
+
+  serveOctoFarmRoutes(app);
+
+  return app;
 }
 
 const logger = new Logger("OctoFarm-Server");
 
 module.exports = {
   setupExpressServer,
   ensureSystemSettingsInitiated,
+  serveOctoFarmRoutes,
```

Comment:
So we can serve the routes for tests. This is called by the serveOctoFarmNormally function, so both can be called.